MIGRATING.md

```diff
@@ -13,15 +13,23 @@ Note: `qiskit-ibm` is not part of the `qiskit` meta package and hence will not b
 1. The `IBMQ` global variable which was an instance of the `IBMQFactory` has been removed.
 1. `IBMQFactory` and `AccountProvider` classes have been removed and the functionality provided by these two classes have been combined and refactored in the new `IBMProvider` class. This class will provide a simplified interface as shown below and serve as the entrypoint going forward.
 
-| Method / Constructor | Description |
-|--------|-------------|
-| IBMProvider.save_account(TOKEN, HUB, GROUP, PROJECT) | Save your account to disk for future use and optionally specify a default provider to return when loading your account. |
-| IBMProvider() | Load account and default provider using saved credentials. |
-| IBMProvider.providers() | List the providers available to your account. |
-| IBMProvider.saved_account() | View the account saved to disk. |
-| IBMProvider.delete_account() | Delete the saved account from disk. |
-| IBMProvider(TOKEN, HUB, GROUP, PROJECT) | Enable your account in the current session and optionally specify a default provider to return. |
-| IBMProvider.active_account() | List the account currently active in the session. |
+   | Method / Constructor | Description |
+   |--------|-------------|
+   | IBMProvider.save_account(TOKEN, HUB, GROUP, PROJECT) | Save your account to disk for future use and optionally specify a default provider to return when loading your account. |
+   | IBMProvider() | Load account and default provider using saved credentials. |
+   | IBMProvider.providers() | List the providers available to your account. |
+   | IBMProvider.saved_account() | View the account saved to disk. |
+   | IBMProvider.delete_account() | Delete the saved account from disk. |
+   | IBMProvider(TOKEN, HUB, GROUP, PROJECT) | Enable your account in the current session and optionally specify a default provider to return. |
+   | IBMProvider.active_account() | List the account currently active in the session. |
+
+1. `IBMBackend.run()` now splits a long list of circuits into multiple jobs and manages them for you, replacing the
```

Comment:
Should we start this line with something like "IBMQBackend has been renamed to IBMBackend." since we don't mention that anywhere before this other than the appendix.